### PR TITLE
T27: Stack<T>

### PR DESCRIPTION
**Pull request by Dakshita-Minocha**

### CODE REVIEW

File: ClassSolution/Classlib/Classlib.csproj [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+	<OutDir>../bin/</OutDir>
```
Comment (meenashriTMM) [inline ClassSolution/Classlib/Classlib.csproj:7]: Spacing is incorrect. Update it for all csproj files

File: ClassSolution/Classlib/MyStack.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
+﻿// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joinees at Metamation, Batch- July 2023.
+// Copyright (c) Metamation India.                                                
+// ------------------------------------------------------------------
+// MyStack.cs
```
Comment (meenashriTMM) [inline ClassSolution/Classlib/MyStack.cs:5]: Do not leave unnecessary spaces at end of statement. Update it for all cs files.

File: Program.cs [hunk 3]
```diff
@@ -3,18 +3,93 @@
 // Copyright (c) Metamation India.
```
Comment (meenashriTMM) [inline Program.cs:3]: No need to change anything in Training solution.

File: ClassSolution/MsTest/TestMyStack.cs [hunk 4]
```diff
@@ -0,0 +1,76 @@
+// ------------------------------------------------------------------------------------------------
```
Comment (meenashriTMM) [inline ClassSolution/MsTest/TestMyStack.cs:1]: Checked if the code coverage is 100%?

**Review by meenashriTMM — CHANGES_REQUESTED** [review]

File: ClassSolution/MsTest/TestMyStack.cs (hunk 4 above)
Comment (Dakshita-Minocha) [reply ClassSolution/MsTest/TestMyStack.cs:1]: yes, its 100%

**Review by meenashriTMM — APPROVED** [review]

**Pull request closed without merging**